docs/docs/using-client-side-only-libraries.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using client side only packages
+issue: https://github.com/gatsbyjs/gatsby/issues/13355
+---
+
+This is a stub. Help our community expand it.
+
+Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
+pull request gets accepted.
+
+On occasion, you may need to use a function or library that only works client side. This usually is because the library in question accesses something that isn't available during server-side rendering (SSR).
```

Comment:
```suggestion
On occasion, you may need to use a function or library that only works client side. This usually is because the library in question accesses something that isn't available during server-side rendering (SSR), like [browser DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model) methods.
```